### PR TITLE
Add timing

### PR DESCRIPTION
**Pull request by JakobWeg**
Updated methodology, documentation, timing, output.

### CODE REVIEW

File: simbev/car.py [hunk 1]
```diff
@@ -67,6 +205,7 @@ def __init__(self, car_type: CarType, number: int, work_parking, home_parking,
         self.region = region
         self.user_spec = 0
         self.hpc_pref = 0
+        self.hpc_data = hpc_data
```
Comment (mosc5) [inline simbev/car.py]: Move this to CarType?

File: simbev/mid_timeseries.py [hunk 2]
```diff
@@ -65,7 +132,7 @@ def get_timeseries(start: datetime.date, end: datetime.date, region, stepsize, d
     # build a matrix containing information about each season during the time span
     while start < end:
         cutoff = get_cutoff(start)
-        if cutoff < end:    # use full season
+        if cutoff < end:    # useful season
```
Comment (mosc5) [inline simbev/mid_timeseries.py]:
```suggestion
        if cutoff < end:    # use the whole season
```

File: simbev/helpers/helpers.py [hunk 3]
```diff
@@ -56,13 +83,154 @@ def export_metadata(
...
+                                                                                         != (-1)].mean(), 4)
+    # save json-file
+    with open(Path(directory, "analysis_mid_{}.json".format(region_id)), "w") as outfile:
+        json.dump(analysis_mid_dict, outfile, indent=4, sort_keys=False)
+
+
+def timeitlog(timing):
+    def decorator(func):
+        os_path = Path(__file__).parent.parent
+        path_to_log_file = Path(os_path, 'results', 'log_file_simbev.txt')
```
Comment (mosc5) [inline simbev/helpers/helpers.py]: I get this error when running:
`FileNotFoundError: [Errno 2] No such file or directory: '/home/moritz/github/simbev/simbev/results/log_file_simbev.txt'`

Probably related to the new structure and missing results directory

**Review by mosc5 — CHANGES_REQUESTED** [review]

File: simbev/car.py (hunk 1 above)
Comment (JakobWeg) [reply simbev/car.py]: Changed it.

File: simbev/helpers/helpers.py (hunk 3 above)
Comment (JakobWeg) [reply simbev/helpers/helpers.py]: That must be the reason I am solving this problem.

**Pull request merged**